src/main/java/hudson/plugins/ansicolor/AnsiColorBuildWrapper.java

```diff
@@ -118,22 +120,26 @@ public boolean configure(final StaplerRequest req, final JSONObject formData) th
                 final List<AnsiColorMap> colorMaps = req.bindJSONToList(AnsiColorMap.class, req.getSubmittedForm().get("colorMap"));
                 for (AnsiColorMap colorMap : colorMaps) {
                     validateFieldName(colorMap.getName());
-                    validateFieldColorLiteral(colorMap.getBlack(), "black");
-                    validateFieldColorLiteral(colorMap.getBlackB(), "blackB");
-                    validateFieldColorLiteral(colorMap.getRed(), "red");
-                    validateFieldColorLiteral(colorMap.getRedB(), "redB");
-                    validateFieldColorLiteral(colorMap.getGreen(), "green");
-                    validateFieldColorLiteral(colorMap.getGreenB(), "greenB");
-                    validateFieldColorLiteral(colorMap.getYellow(), "yellow");
-                    validateFieldColorLiteral(colorMap.getYellowB(), "yellowB");
-                    validateFieldColorLiteral(colorMap.getBlue(), "blue");
-                    validateFieldColorLiteral(colorMap.getBlueB(), "blueB");
-                    validateFieldColorLiteral(colorMap.getMagenta(), "magenta");
-                    validateFieldColorLiteral(colorMap.getMagentaB(), "magentaB");
-                    validateFieldColorLiteral(colorMap.getCyan(), "cyan");
-                    validateFieldColorLiteral(colorMap.getCyanB(), "cyanB");
-                    validateFieldColorLiteral(colorMap.getWhite(), "white");
-                    validateFieldColorLiteral(colorMap.getWhiteB(), "whiteB");
+                    final HashMap<String, Supplier<String>> validatedFields = new HashMap<>();
```

Comment:
I think this belongs in some other static method that returns the map of validated fields.

Reply:
Did that and covered with tests.